ext/sqlite3/tests/bug68760.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+Bug #68760 (Callback throws exception behaviour.)
```

Comment:
This needs to the standard SKIPIF section
--SKIPIF--
<?php require_once(dirname(__FILE__) . '/skipif.inc'); ?>